scanpipe/pipelines/deploy_to_develop.py

```diff
@@ -29,9 +29,10 @@
 from scanpipe.pipes import matchcode
 from scanpipe.pipes import purldb
 from scanpipe.pipes import scancode
+from scanpipe.pipes.d2d_config import DefaultEcosystemConfig
 
 
-class DeployToDevelop(Pipeline):
+class DeployToDevelop(Pipeline, DefaultEcosystemConfig):
```

Comment:
I think we should make this work without the need for the extra mixing. See further suggestions.

Reply:
I have made this a pipeline attribute instead now. Thanks